src/laboratory-resource.ts

```diff
@@ -88,3 +90,12 @@ export function rejectLabOrder(orderId: string, comment: string, abortController
     },
   });
 }
+
+const usePatientAge = (uuid: string) => {
```

Comment:
Let's do some clean up and delete this since it's no longer necessary

Reply:
Resolved